compiler/damlc/daml-prim-src/GHC/Classes.daml

```diff
@@ -339,6 +344,14 @@ instance Ord Decimal where
     (>)  = primitive @"BEGreater"
 #endif
 
+#ifdef DAML_UNSTABLE
+instance Ord BigDec where
```

Comment:
FWIW I think we should rather define this instance only on the user-facing `BigDecimal` directly using the `CompareBigDec` there